stickers.yml

```diff
@@ -23934,7 +23934,7 @@ da1a3e38bda5922f2c6e96e04996a103:
     - Hong Kong
     - hkg
 c4acf433026191d040ac64361bf0388a:
-  key: 2db788e72a671dcffc3dfcca9c4b1a917e986f1686368376684cf6d6a1eb08022
```

Comment:
FYI While adding length restrictions to the schema (keys should be 64 characters long) I caught this pack that had a 65-character long key. Apparently the last `2` was not required, and the Signal API seems to have just ignored it. I tested this pack by removing the extra digit and it loaded successfully.